### PR TITLE
Update ore-loot-tables.md with xp method for non experimental

### PR DESCRIPTION
**Pull request by Keyyard**
I have introduced a simple method to obtain experience orbs (exp orbs) from blocks without needing to activate the experimental mode. This method only requires basic knowledge of working with blocks, such as creating and using non-experimental blocks,. It allows them to drop exp orbs. It is particularly useful for non-experimental projects and marketplace.

### CODE REVIEW

**Review by Keyyard — COMMENTED** [review]
I have fixed the language part for better merging

File: docs/blocks/ore-loot-tables.md [hunk 1]
```diff
@@ -121,12 +122,85 @@ Also note that it can correctly detect only 1st and 2nd enchantment level.
...
+```
+
+2. Create a function to loop through and process the dropped items:
+
+```
+execute as @e[type=item,name="Raw Silver",tag=!xp] at @s run structure load ore_xp_reward ~~~
+execute as @e[type=item,name="Raw Silver",tag=!xp] run tag @s add xp
+```
+
+This function will execute for any item entity with the name "Raw Silver", which is ``wiki:raw_silver`` item ID that does not have the "xp" tag. It loads the "my_xp_structure" at the item's location and then adds the "xp" tag to the item.
```
Comment (QuazChick) [inline docs/blocks/ore-loot-tables.md]: Change `my_xp_structure` to `ore_xp_reward`.

File: docs/blocks/ore-loot-tables.md [hunk 2]
```diff
@@ -121,12 +122,85 @@ Also note that it can correctly detect only 1st and 2nd enchantment level.
...
+In this case, we add an existing item called "minecraft:barrier" as a dummy item to trigger the XP drop, we can create a in-existing item if you want.
+
+2. Next, create a function to loop through and process the dropped items:
+
+```
+execute as @e[type=item,name="Barrier"] at @s run structure load my_xp_structure ~~~
+execute as @e[type=item,name="Barrier"] run kill @s
+```
+
+This function will execute for any item entity with the name "Barrier" (our dummy item). It loads a structure called "my_xp_structure" at the item's location and then kills the item.
```
Comment (QuazChick) [inline docs/blocks/ore-loot-tables.md]: Change `my_xp_structure` to `ore_xp_reward`.

File: docs/blocks/ore-loot-tables.md [hunk 3]
```diff
@@ -121,12 +122,85 @@ Also note that it can correctly detect only 1st and 2nd enchantment level.
...
+  ]
+}
+```
+
+In this case, we add an existing item called "minecraft:barrier" as a dummy item to trigger the XP drop, we can create a in-existing item if you want.
+
+2. Next, create a function to loop through and process the dropped items:
+
+```
+execute as @e[type=item,name="Barrier"] at @s run structure load my_xp_structure ~~~
```
Comment (QuazChick) [inline docs/blocks/ore-loot-tables.md]: Change `my_xp_structure` to `ore_xp_reward`.

**Review by QuazChick — CHANGES_REQUESTED** [review]
Some instances of `my_xp_structure` remain unchanged.

**Pull request merged**